src/transaction/flashback_cl.c

```diff
@@ -774,7 +805,7 @@ flashback_print_delete (char **loginfo, int trid, char *user, const char *classn
 	}
       else
 	{
-	  strcat (sql, " )");
+	  strcat (sql, " );");
```

Comment:
Doesn't this affect the length of this sql?

Reply:
yes, 
`error = flashback_check_and_resize_sql_memory (&sql, strlen (sql) + 3, &max_sql_size);`

it means the maximum value among the lengths of the two strings (", " or " );" ), and 3 is sufficient.

Oh.. no, it is not sufficient, I will change the `strlen (sql) + 3` to `strlen (sql) + 4` . 

Thank you

Reply:
Or remove space in front of ).